### PR TITLE
Split troubleshooting file

### PR DESCRIPTION
**Pull request by mjivraja**
The purpose to split this file is downstream inclusion. Also, there was more than one heading per file which is not a standard modularization practice. In addition to this, it will help to review the diff files more conveniently. Therefore, it became necessary to split this file, addressing many purposes.

* [X] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4 on EL8 only)
* [X] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [X] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.4 or older.

### CODE REVIEW

File: guides/common/attributes-foreman-el.adoc [hunk 1]
```diff
@@ -1,4 +1,6 @@
-// Overrides for foreman-el build
+// Overrides for foreman-el build\
```
Comment (Lennonka) [inline guides/common/attributes-foreman-el.adoc:1]:
```suggestion
// Overrides for foreman-el build
```

**Review by Lennonka — COMMENTED** [review]
However, should a SME ack that the modules are correctly used in these older Foreman versions as you added them?

File: guides/common/modules/ref_troubleshooting-sync-errors.adoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+= Troubleshooting Sync Errors
+
+"[Errno 1] Operation not permitted: ..." during repository syncing::
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+# chown --recursive pulp.pulp /var/lib/pulp/media/
+----
+
+"{“policy”:[""" is not a valid choice."]}" during Debian repository syncing::
```
Comment (ehelms) [inline guides/common/modules/ref_troubleshooting-sync-errors.adoc:11]: This should be Katello / Orcarhino only.

Comment (ehelms) [reply guides/common/modules/ref_troubleshooting-sync-errors.adoc:11]: Oh I see this whole file is scoped in the table of contents.

**Review by ehelms — APPROVED** [review]
@evgeni could you give this a glance as well?

File: guides/common/modules/con_troubleshooting-ruby.adoc [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+In that case, you have to resolve dependencies manually as follows:
+
+List the enabled modules:
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf module list --enabled
+----
+
+If the Ruby 2.7 module has already been enabled, perform a module reset:
```
Comment (evgeni) [inline guides/common/modules/con_troubleshooting-ruby.adoc:14]:
```suggestion
If the Ruby 2.5 module has already been enabled, perform a module reset:
```

Comment (evgeni) [reply guides/common/modules/con_troubleshooting-ruby.adoc:14]: (this issue is also present in the other branches and should be fixed there too!)

**Review by evgeni — CHANGES_REQUESTED** [review]
ACK pending the Ruby version fix.

File: guides/common/modules/con_troubleshooting-ruby.adoc (hunk 3 above)
Comment (evgeni) [reply guides/common/modules/con_troubleshooting-ruby.adoc:14]: https://github.com/theforeman/foreman-documentation/pull/2257

**Comment by Lennonka** [comment]
Please, remember to file another PR for the Ruby version fix for 3.3+ and master.

**Review by Lennonka — APPROVED** [review]
LGTM, merging

**Comment by Lennonka** [comment]
There's a conflict on 3.2 and I expect it on 3.1 as well. Please, file another PR.

**Comment by mjivraja** [comment]
> There's a conflict on 3.2 and I expect it on 3.1 as well. Please, investigate and eventually, file another PR.

Ack! @Lennonka Thanks :star:

**Pull request merged**